Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/Pass.h

```diff
@@ -570,6 +579,9 @@ namespace AZ
             // buffers and images don't get deleted during attachment build phase
             AZStd::vector<Ptr<PassAttachment>> m_importedAttachmentStore;
 
+            // List of connections on this pass that will be registered with the pipeline for reference in a global manner
+            PipelineConnectionList m_pipelineConnections;
```

Comment:
Same nit - for clarity I'd call it m_globalConnections; it is much clearer even without any comment.